src/renderers/dom/client/ReactBrowserEventEmitter.js

```diff
@@ -359,8 +366,12 @@ var ReactBrowserEventEmitter = Object.assign({}, ReactEventEmitterMixin, {
    */
   ensureScrollValueMonitoring: function() {
     if (hasEventPageXY === undefined) {
-      hasEventPageXY =
-        document.createEvent && 'pageX' in document.createEvent('MouseEvent');
+      if (document.createEvent) {
+        var evt = document.createEvent('MouseEvent');
+        hasEventPageXY = evt !== null && 'pageX' in evt;
```

Comment:
This would need to be `!= null` instead of `!== null`. If we’re protecting, let’s protect fully.
You will also need to add a note like

```js
// Protect against document.createEvent() returning null
// Some popup blocker extensions appear to do this:
// https://github.com/facebook/react/issues/6887
```